app/assets/javascripts/modules/media_viewer.js

```diff
@@ -180,9 +220,11 @@
       init: function() {
         setupThumbSlider();
         authCheck();
-        this.initializeDashPlayer();
       },
 
+      // This is currently unused under video-js.
+      // We should remove it (as well as all related code)
+      // if we go w/ video-js as our media client
```

Comment:
if it is totally unused, i think i'd be in favor of removing it now, unless you think we'd need it back soon anyway (esp if it'd be a pain to add back).